modules/api/pages/bpm-api.adoc

```diff
@@ -4553,6 +4553,7 @@ Mandatory filters: connector_process_id, connector_name, connector_version
 ]
 ----
 
+[#connector-failure]
```

Comment:
I know but IntelliJ tells me it's better to had explicit anchors when referring to them. What is the best practice on that point?

Reply:
I don't really what could be the best practice, IMHO when the generated anchor is short (i.e the title is short) use it, it avoids to add code.
If the title is too long then creating a custom anchor can be nice since the anchor appears in the final URL.

Reply:
Is there something that checks for broken links?
One. benefit of putting a that here is also to remember that we have a link and if we rename it we broke something. Otherwise, yep it's only noise.
I'll remove it